build-tools/scripts/BuildEverything.mk

```diff
@@ -164,8 +170,22 @@ package-oss $(ZIP_OUTPUT):
 		for f in `cat $$_sl` ; do \
 			echo "$(ZIP_OUTPUT_BASENAME)/bin/$$c/lib/xamarin.android/xbuild/$$f" >> "$$_exclude_list"; \
 		done; \
-	done; \
+	done
+ifeq ($(ZIP_EXTENSION),zip)
 	zip -r "$(ZIP_OUTPUT)" \
 		`ls -1d $(_BUNDLE_ZIPS_INCLUDE) 2>/dev/null` \
 		"-x@$$_exclude_list"
+else ifeq ($(ZIP_EXTENSION),tar.bz2)
+	tar $(_BUNDLE_ZIPS_EXCLUDE:%=--exclude=%) -cjhvf "$(ZIP_OUTPUT)" `ls -1d $(_BUNDLE_ZIPS_INCLUDE) 2>/dev/null`
```

Comment:
Turns out there's an `--exclude-from` flag to `tar` I'd missed, which takes an exclude file from the command like (like `-x@` for zip), so that's unified now.